packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -130,7 +130,12 @@ const merge = (a, b) => deepmerge(a, b, {arrayMerge: (orignal, replacement) => r
 const runGenerator = (answers, {outputDir, verbose}) => {
     checkOutputDir(outputDir)
     // Excluding pwa-kit-create-app, these are the public pwa-kit-* packages that can be installed through NPM.
-    const npmInstallables = ['pwa-kit-react-sdk', 'pwa-kit-dev', 'pwa-kit-runtime']
```

Comment:
Please ignore any changes to the files in `pwa-kit-create-app`. They're just for my own testing purpose. I plan to undo the changes.

Reply:
@vmarta did you revert these now that we're aiming to finalize and merge all of the various workstreams in `feature/template-extensibility`?